linux_pipeline/Jenkinsfile_developer_comparison_validation

```diff
@@ -35,66 +51,113 @@ properties ([
                         string(name: 'LISA_TEST_SUITE', value: "${LISA_TEST_SUITE}"),
+                        string(name: 'LISA_PERF_TEST_XML', value: "${perfXml[LISA_PERF_TEST]}"),
+                        string(name: 'LISA_PERF_OPTIONS', value: "${PERF_OPTIONS}"),
                         string(name: 'ENABLED_STAGES', value: "${ENABLED_STAGES}")], propagate: false;
                 }
             }
             parallel runs
-    
     }
+
     stage ("aggregate_results") {
         dir ('comparison_results' + env.BUILD_NUMBER) {
-            copyArtifacts(projectName: 'pipeline-developer-patch-validation/master', selector: specific("${patchedRun.number}"), target: "patched_run");
+            copyArtifacts(projectName: 'pipeline-developer-patch-validation/PR-200', selector: specific("${patchedRun.number}"), target: "patched_run");
             archiveArtifacts 'patched_run/**';
-            
             if (env.SKIP_UNPATCHED == 'no') {
-                copyArtifacts(projectName: 'pipeline-developer-patch-validation/master', selector: specific("${unpatchedRun.number}"), target: "unpatched_run");
+                copyArtifacts(projectName: 'pipeline-developer-patch-validation/PR-200', selector: specific("${unpatchedRun.number}"), target: "unpatched_run");
                 archiveArtifacts 'unpatched_run/**';
-                stash includes: 'patched_run/lis-test/WS2012R2/lisa/TestResults/**/Report*.xml', name: 'patchedReport'
-                stash includes: 'unpatched_run/lis-test/WS2012R2/lisa/TestResults/**/Report*.xml', name: 'unpatchedReport'
+                stash includes: 'patched_run/*-functional/TestResults/**/Report*.xml', name: 'patchedReport'
+                stash includes: 'unpatched_run/*-functional/TestResults/**/Report*.xml', name: 'unpatchedReport'
+                stash includes: 'patched_run/*-perf/lis-test/WS2012R2/lisa/TestResults/', name: 'patchedPerf'
+                stash includes: 'unpatched_run/*-perf/lis-test/WS2012R2/lisa/TestResults/', name: 'unpatchedPerf'
+                stash includes: 'unpatched_run/scripts/package_building/kernel_versions.ini', name: 'unpatchedMetadata'
             }
         }
     }
+
     stage ("compare_results") {
         if (env.SKIP_UNPATCHED == 'no') {
             dir('comparison_results' + env.BUILD_NUMBER) {
+                sh 'rm -rf *'
                 checkout scm
                 unstash 'patchedReport'
                 unstash 'unpatchedReport'
+                unstash 'patchedPerf'
+                unstash 'unpatchedPerf'
+                unstash 'unpatchedMetadata'
+                sh '''#!/bin/bash
+                    INI_PATH="./unpatched_run/scripts/package_building/kernel_versions.ini"
+                    echo '[METADATA]' > ./metadata.ini
+                    KERNEL="$(crudini --get $INI_PATH KERNEL_BUILT folder)"
+                    KERNEL="${KERNEL%-unpatched*}"
+                    echo "KERNEL_VERSION = $KERNEL" >> ./metadata.ini
```

Comment:
use crudini instead of this code:
crudini --set "./metadata.ini" 'METADATA' KERNEL_VERSION $KERNEL